packages/s-deploy/src/deploy.ts

```diff
@@ -0,0 +1,59 @@
+import { lodash as _, Logger, fs } from '@serverless-cd/core';
+import execa from 'execa';
+
+export interface IProps {
+  resource?: string; // Specify a resource in s.yaml. Use the command to s xxx deploy.
+  deployFile?: string; // Specific yaml file. Use the command to s deploy -t xxx 
+  deployArgs?: string[]; // Specific custom args. Merge with other command args during deployment.
+  envName?: string; // Specify envionment. Use the command to s deploy --env
+  debugMode?: boolean; // Use the command to s deploy --debug
+  worksapce?: string // worksapce dir.
```

Comment:
duplicate: IMHO workspace is duplicate with `cwd`.

refer to: https://serverless-cd.cn/docs/dev-guide/toolkit/engine/#stepoptions
![image](https://github.com/serverless-cd/plugins/assets/16112202/415fc063-6d01-472d-8f0b-88d45e9492e7)

Reply:
typo: workspace

Reply:
> duplicate: IMHO workspace is duplicate with `cwd`.
> 
> refer to: https://serverless-cd.cn/docs/dev-guide/toolkit/engine/#stepoptions ![image](https://user-images.githubusercontent.com/16112202/275775410-415fc063-6d01-472d-8f0b-88d45e9492e7.png)

working-director cannot be obtained globally, and engine does not modify process.cwd. Therefore, I need to record this value from the context when constructing input.